src/main/java/uk/gov/hmcts/ccd/data/roleassignment/RoleAssignmentRecord.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.ccd.data.roleassignment;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Builder
+@Data
+public class RoleAssignmentRecord {
```

Comment:
Can we change the name to RoleAssignment, instead of Record postfix

Reply:
Following https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1386808550 and we map it to the RoleAssignment in our model.

Reply:
can we follow the naming conventions we use in Data Store? I think we call these response objects with suffix `Response`?  So `RoleAssignmentResponse`?
Can't recall now what the convention is but let's follow what we have

Reply:
Will use RoleAssignmentResponse for the top level object and DTO suffix for its children. Just to distinguish between the objects coming from the API and our model.